finetuner/tuner/augmentation.py

```diff
@@ -16,22 +17,61 @@ def vision_preprocessor(
     :param width: image width.
     :param default_channel_axis: The color channel of the input image, by default -1, the expected input is H, W, C.
     :param target_channel_axis: The color channel of the output image, by default 0, the expected output is C, H, W.
+    :param phase: phase of experiment, either `train` or `validation`. At `validation` phase, will not apply
+      random transformation.
     """
 
     def preprocess_fn(doc):
         return _vision_preprocessor(
-            doc, height, width, default_channel_axis, target_channel_axis
+            doc, height, width, default_channel_axis, target_channel_axis, phase
         )
 
     return preprocess_fn
 
 
+def _set_image_blob_normalization(
```

Comment:
This can actually also be done with albumentations (for example [`Normalize`](https://albumentations.ai/docs/api_reference/augmentations/transforms/#albumentations.augmentations.transforms.Normalize) and [`ToFloat`](https://albumentations.ai/docs/api_reference/augmentations/transforms/#albumentations.augmentations.transforms.ToFloat)). Wouldn't it be better to use that?